src/activities/categorization/resource/board/category_odd_even.qml

```diff
@@ -23,38 +23,38 @@ import QtQuick 2.0
 
 QtObject {
     property bool isEmbedded: true
-    property string imagesPrefix: "qrc:/gcompris/src/activities/categorization/resource/images/odd_even/"
+    property string imagesPrefix: "qrc:/gcompris/src/activities/lang/resource/words_sample/"
     property variant levels: [
         {
             "type": "lesson",
             "name": qsTr("odd even numbers"),
-            "image": imagesPrefix + "numbers.jpg",
+            "image": imagesPrefix + "eleven.png",
             "content": [
                 {
                     "instructions": qsTr("Place the EVEN Numbers to the right and ODD Numbers to the left"),
-                    "image": imagesPrefix + "even2.jpg",
-                    "maxNumberOfGood": 4,
+                    "image": imagesPrefix + "sixteen.png",
+                    "maxNumberOfGood": 5,
                     "maxNumberOfBad": 4,
```

Comment:
As mentioned use more images in initial levels (teaching part) . Add 12 images in level1, 9 images in level2, 6 images in level 3 (concept building) , 6 images in level 4 (testing).